cmd/credentialUtil.go

```diff
@@ -135,17 +135,62 @@ func GetOAuthTokenManagerInstance() (*common.UserOAuthTokenManager, error) {
 // 4. If there is OAuth token info passed from env var, indicating using token credential. (Note: this is only for testing)
 // 5. Otherwise use anonymous credential.
 // The implementation logic follows above rule, and adjusts sequence to save web request(for verifying public resource).
-func getBlobCredentialType(ctx context.Context, blobResourceURL string, canBePublic bool, standaloneSAS bool, cpkOptions common.CpkOptions) (common.CredentialType, bool, error) {
+func getBlobCredentialType(ctx context.Context, blobResourceURL string, canBePublic bool, standaloneSAS string, cpkOptions common.CpkOptions) (common.CredentialType, bool, error) {
 	resourceURL, err := url.Parse(blobResourceURL)
 
 	if err != nil {
 		return common.ECredentialType.Unknown(), false, errors.New("provided blob resource string is not in URL format")
 	}
 
+	if standaloneSAS != "" {
```

Comment:
What is the default of standaloneSAS? If it is empty, what will be the value of RawQuery?
`resourceURL.RawQuery = standaloneSAS`